components/Fields/Field.vue

```diff
@@ -230,8 +230,9 @@ export default defineNuxtComponent({
         v-else-if="isWebsite"
         :key="`website_${item}`"
       >
-        <ExternalLink :href="item">
-          {{ item }}
+        <ExternalLink :title="item" :href="item">
+          {{ context !== 'label_list' ? item : '' }}
+          <br>
```

Comment:
Why a `<br>` here ? And not for other array properties ? Eg `phone` is already show one item per line with requiring `<br>`.

Reply:
It was for better readability when we display only icons. But I have to admit that it's not a standardized way to do ...
I think i could be managed later, when we will work on Field UI more globally.